src/visual.ts

```diff
@@ -167,7 +167,7 @@ module powerbi.extensibility.visual {
             }
 
             let categoricalValues: BulletChartColumns<any[]> = BulletChartColumns.getCategoricalValues(dataView);
-            let settings: BulletchartSettings = BulletChart.parseSettings(dataView, categorical.Category.source, colorHelper);
+            let settings: BulletchartSettings = BulletChart.parseSettings(dataView, colorHelper);
```

Comment:
Code looks good but we should update package version and cover change in CHANGELOG